packages/fast-components-styles-msft/src/subheading/index.ts

```diff
@@ -0,0 +1,27 @@
+import { IDesignSystem } from "../design-system";
+import { ComponentStyles, ICSSRules } from "@microsoft/fast-jss-manager";
+import { ISubheadingClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+import { fontWeight } from "../utilities/fonts";
+
+const styles: ComponentStyles<ISubheadingClassNameContract, IDesignSystem> = {
+    subheading_1: {

```

Comment:
the `ISubheadingClassNameContrast` should expose a `subheading` class that all instances of subheading get. Then, fontWeight can just be applied to that class.